Gemfile.lock

```diff
@@ -0,0 +1,218 @@
+GEM
```

Comment:
Should we commit this file?

Reply:
I committed it mainly to enable caching gems in the pipeline. I checksum Gemfile.lock for the cache key. Could probably go off just the Gemfile though, especially since I'm only using those gems in the pipeline itself